qiskit_nature/second_q/mappers/qubit_mapper.py

```diff
@@ -61,6 +104,53 @@ def map(self, second_q_op: SparseLabelOp) -> PauliSumOp:
         """
         raise NotImplementedError()
 
+    def map(
+        self,
+        second_q_ops: SparseLabelOp | PauliSumOp | ListOrDictType[SparseLabelOp | PauliSumOp],
```

Comment:
The same question applies here: why the `PauliSumOp`?

If it is really needed, we should attempt to use `SparsePauliOp` instead since the goal is to get
rid of opflow objects.

Reply:
As discussed, I rolled back these changes